### ISSUE TITLE
Unrecognized capability with ChromeOptions

### ISSUE THREAD

**Issue by owickstrom** [nbloomf/webdriver-w3c#21]
Hi! Thanks for this library. I'm currently using it in [quickstrom.io](https://quickstrom.io).

When using chromedriver and configuring this client with `ChromeOptions`, I get the following error:

```
$ chromedriver --port=4444 --verbose
Starting ChromeDriver 83.0.4103.39 (ccbf011cb2d2b19b506d844400483861342c20cd-refs/branch-heads/4103@{#416}) on port 4444
Only local connections are allowed.
Please see https://chromedriver.chromium.org/security-considerations for suggestions on keeping ChromeDriver safe.
ChromeDriver was started successfully.
[1598979125.870][INFO]: [2f6e04433a153a576dcbbf0f903187ff] COMMAND InitSession {
   "capabilities": {
      "alwaysMatch": {
         "browserName": "chrome",
         "chromeOptions": {
            "binary": "google-chrome-stable"
         }
      }
   },
   "desiredCapabilities": {
      "browserName": "chrome",
      "chromeOptions": {
         "binary": "google-chrome-stable"
      }
   }
}
[1598979125.874][INFO]: [2f6e04433a153a576dcbbf0f903187ff] RESPONSE InitSession ERROR invalid argument: unrecognized capability: chromeOptions
[1598979125.874][DEBUG]: Log type 'driver' lost 1 entries on destruction
```

Should it be `goog:chromeOptions`?

**Comment by nbloomf** [comment]
Thanks for the bug report! I'll have to look into this to see exactly what's going on, but off the top of my head I suspect this has to do with changes to chromedriver that have happened since support was added to this library (CI for this project is using a pretty old version of chromedriver).

**Comment by owickstrom** [comment]
Gotcha. I'll try setting `goog:chromeOptions` using `newSession'` as you noted in #22 , and see if that works better.

**Comment by owickstrom** [comment]
Yep, got it running!

**Comment by owickstrom** [comment]
```haskell
newSession' addCaps someCaps
where
    addCaps :: JSON.Value -> JSON.Value
    addCaps =
      key "capabilities" . key "alwaysMatch" . _Object
        %~ ( <>
               [ ( "goog:chromeOptions",
                   JSON.Object
                     [ ("binary", "/nix/store/wv5f8h1m3wjjcgd20sf13x10bwyvf4ms-google-chrome-84.0.4147.105/bin/google-chrome-stable")
                     , ("args", JSON.Array ["headless"])
                     ]
                 )
               ]
           )
```

**Comment by nbloomf** [comment]
That's great! I'll leave this issue open until I get a minute to add this example to the documentation. :)

**Comment by owickstrom** [comment]
Nice!

**Comment by nbloomf** [comment]
Just following up here - from the [release notes](https://chromedriver.storage.googleapis.com/75.0.3770.8/notes.txt) it looks like chromedriver started prefixing this key in version 75 for compliance with the w3c spec. This is fixed by #27.
